theseus/geometry/se3.py

```diff
@@ -261,34 +261,8 @@ def transform_to(
         jacobians: Optional[List[torch.Tensor]] = None,
     ) -> Point3:
```

Comment:
@mhmukadam Do we want to make a similar (future) change on `theseus`? If so, I can make this in another PR so that we add a deprecation message and an alias for the new names.